### PR TITLE
User Story 12 - Interview full article text edit and styling

### PR DESCRIPTION
**Pull request by IC-Socaciu**
I have tried to add a minimal style to the full articles(5 out of 6 are done), added some background for the questions and some links that lead to external pages. For this PR, please, use the first interview as it has all of the elements that I have styled. Feel free to add any suggestions and let me know if the dividers and colour backgrounds make the text easier to read. Thank you.

### CODE REVIEW

File: public/interviews.js [hunk 1]
```diff
@@ -412,175 +412,273 @@ filter: drop-shadow(0px 0px 5px rgba(0, 0, 0, 0.25));"/>
...
+The cover art is signed by [**Merliis Järve**](https://www.instagram.com/merliisart/?hl=de) and the album design was realized by [**Maris Savik**](https://www.instagram.com/preilimaris/?hl=de).
 
-*As always, a huge **Thank You** for your time, openness and availability!*
+In recent news, [**Toomas Vanem**](https://www.facebook.com/ToomasGuitar/‚) has been invited to play on festival [**SÕRU JAZZ FESTIVAL**](https://www.facebook.com/sorujazz), which will be held on 18.06.2022 on the Estonian island of Hiiumaa.
 
 
+*As always, a huge **Thank You** for your time, openness and availability!*
 
-**AAP – Hello, Toomas! Welcome on our virtual pages and thank you for taking the time to do this interview with All-Access-Pass!**
+<hr style="background-color: #dfdfdf; height: 2px;box-shadow: 0px -5px 10px rgba(0, 0, 0, 0.25); width: 100px;
```
Comment (jonas8900) [inline public/interviews.js:443]: could you explain what hr is?

Comment (jonas8900) [reply public/interviews.js:443]: i do not really understand how this kind of styling works, i think this is inline-styling isn't it?

Comment (jonas8900) [reply public/interviews.js:443]: we saw you use the text:`to place your interview here with inlinestyling`.

**Comment by jonas8900** [comment]
i like your app idea and the look from your app a lot ! The interview.js is very huge and we are not sure if we should check the content. We are a bit confused because of the inline styling, but with this huge content we don't know a better way to style parts of the content.

File: public/interviews.js (hunk 1 above)
Comment (IC-Socaciu) [reply public/interviews.js:443]: Thank you for the review ! Yes the data that I use is quite voluminous and the easiest way to edit my array object containing the interviews is to add the style directly to the interview texts. The <hr> for instance adds the styled divider that appears each few interviews. This all also goes hand in hand with the 'import Markdown from "markdown-to-jsx";' from the [interviewsId]. :)

**Pull request merged**